SummerProject/static/src/components/Footer/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
+
+class Footer extends React.Component {
```

Comment:
FYI: You can simplify this by using a function-component - https://reactjs.org/docs/components-and-props.html#function-and-class-components